packages/modal/src/components/drawer/Drawer.module.css

```diff
@@ -6,29 +6,64 @@
 
   .content {
     box-shadow: var(--swui-shadow-modal);
+    background: white;
```

Comment:
Always hardcoded white?

Reply:
Yes!

:) 

<img width="603" alt="image" src="https://user-images.githubusercontent.com/1266041/161514202-b3f0250f-56a7-4327-80c8-a19d4e836154.png">

Reply:
The hardcoded value is only a fallback if prop `background` is not used. `background`-prop overrides this value.